### DISCUSSION TITLE
Two different accounts, two different RPM UI interfaces, two different models (in structure) ?

### DISCUSSION THREAD

**Discussion by JanFromNaomiAnimation** [readyplayerme/rpm-unity-sdk-core#56] [Q&A] [answered]
Can you please help me understand how it works? When I log in via the web using account 1, I see a white interface and a different avatar builder compared to when I log in using account 2. Additionally, the models' structures are different between the two accounts. Account 1 has a single mesh that contains all expected face blendshapes (more than 50), while account 2 has more meshes but only two blendshapes on the face.

![image](https://user-images.githubusercontent.com/124244732/226066782-fa21a93a-cb6c-459b-8965-cdbf46e2c44d.png)

**Comment by srcnalt** [comment, marked as answer]
Hi @JanFromNaomiAnimation, partners with subdomains can set a default avatar config to how they want to receive an avatar by default. So they can get an avatar with this setting when they do not use an Avatar Config Scriptable Object in Unity. I your case these defaults might be different.

As a partner (subdomain owner) you can use the same domain, but multiple Avatar Config Scriptable Objects that are set in the Settings window, to create multiple different quality avatars.

**Comment by JanFromNaomiAnimation** [comment]
Thnx @srcnalt for the quick response. We will try to create an explicit avatar configuration to fix our problem with blendshapes. However, I couldn't find any property (even in documentation) or parameter that sets or indicates whether the mesh will be divided into multiple game objects (like in account2, which has 9 gameObjects with separated skinned mesh renderers) or into a single Renderer_Avatar mesh (like in account1)

**Reply by srcnalt** [reply]
Mesh division is related to texture atlasing ability. If the texture is atlased then avatar will be received as a single mesh and single material.
You can find more details here: https://docs.readyplayer.me/ready-player-me/api-reference/avatars/full-body-avatars

**Comment by JanFromNaomiAnimation** [comment]
Ooooh, I'm so sorry I had to miss that. Thnx a lot for your time and help in such a short time. It means a lot.
